### ISSUE TITLE
If MEDIA_ROOT declared as absolute path, it deleting everything in MEDIA_ROOT

### ISSUE THREAD

**Issue by vartagg** [akolpakov/django-unused-media#10]
* If `MEDIA_ROOT` declared as **absolute path**, it **deleting everything** in `MEDIA_ROOT`
* `remove_empty_dirs` should be optional command argument
* Incorrect output in confirmation (`y`)
* Django 1.11 should be presented as supporting version because the app works well with it

My PR with fixes #9

**Comment by beaumartinez** [comment]
Trying to install with Django 1.11 actually uninstalls that version, and installs 1.10.7 instead.

**Comment by akolpakov** [comment]
@beaumartinez Working with Django 1.11 was fixed by #11 in version 0.1.7, available in PyPi

**Comment by akolpakov** [comment]
@vartagg Thanks for contribution!

Based on your MR, I made some fixes.

- Django 1.11 fixed by #11 
- Incorrect output in confirmation (y) - now it accepts case insensitive values
- Optional removing of empty dirs will be done based on your MR in #12
- This case will be only for issue with absolute path. Just copy comments from MR:

>And plugin is working with absolute path, even in tests absolute path is using.

Yes, it is. But unfortunately, tests do not cover this bug. If you test with at least such parameters:
* Python 3.6 (maybe don't working on older v3 Python versions, but I had this one when the bug was reproduced)
* Files located in subdirectories like 
`images/1/2017/image.jpg`
`audio/10/2016/foo.mp3`
* MEDIA_ROOT declared as absolute path (already presented in your settings), **but not in** `django application directory` or its subdirectories.

I sure that the last is a key parameter, because I saw in Debugger when `images/1/2017/image.jpg` (stored in `fileobj.url`) is comparing with something like `../../../images/1/2017/image.jpg` ("relative path" of file provided by `os.path` module when `MEDIA_ROOT` path was higher then django application directory in filesystem tree) so comparison function didn't detect this because `../../../images/1/2017/image.jpg` is not a real path bound with some django file field object. So, everything in `MEDIA_ROOT` (after confirmation and on test environment, what is good) is deleted.
